spinn_front_end_common/interface/provenance/provenance_writer.py

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 import logging
-from spinn_utilities.config_holder import get_config_int_or_none
+from spinn_utilities.config_holder import (
```

Comment:
It would be interesting to test this hypothesis with the latest code.  The issue I have seen is that writing reports is very slow when the disk is not very fast (this can happen with multiple people writing at the same time for example).  This happens on the tests in particular which is why this was done.  It may be worth extracting these changes to another branch though as they are different and could be done if any of the tests are kept.